src/components/SQFormDialog/SQFormDialogInner.js

```diff
@@ -40,54 +40,96 @@ function SQFormDialogInner({
   saveButtonText,
   shouldRequireFieldUpdates = false,
   title,
-  muiGridProps
+  muiGridProps,
+  showCancelConfirmDialog,
```

Comment:
We need to add `showCancelConfirmDialog` and `cancelConfirmDialogProps` in `SQFormDialogInner.propTypes`.

Reply:
Sure @umerjamal100 Thank you